readme.html

```diff
@@ -31,6 +31,13 @@
 	</ul>
 </p>
 
+<p><b>Detonate File: Filename Parameter</b></p>
+  <ul>
+    <li>According to the Wildfire documentation: "When submitting supported script files, you must specify an accurate filename using the context parameter, otherwise WildFire is unable to parse the file and returns a 418 Unsupported File Type response."</li>
+    <li>Please see the following link for more information: <a href="https://docs.paloaltonetworks.com/wildfire/u-v/wildfire-api/submit-files-and-links-through-the-wildfire-api/submit-a-remote-file-to-wildfire-api.html" target="_blank">Wildfire API Documentation</a></li>
+  </ul>
+</p>
+
```

Comment:
Do we also need to update `readme.md` as well?

Reply:
you can just update `readme.html`

[start-release](https://github.com/splunk-soar-connectors/wildfire/actions/workflows/start-release.yml) will update `readme.md` from the updated `readme.html`

Reply:
ah thats what I initially thought. welp i updated it anyway :O

Reply:
ah I see. Thanks. Resolving this thread.